packages/server/src/service-worker-manager.ts

```diff
@@ -174,19 +186,30 @@ export class ServiceWorkerManager implements IServiceWorkerManager {
   /**
    * Handle a message received on the BroadcastChannel
    */
-  private _onBroadcastMessage = async <T extends TDriveMethod>(
+  private _onBroadcastMessage = async (
     event: MessageEvent<{
-      data: TDriveRequest<T>;
+      data: any;
```

Comment:
Is there a way to keep a proper type for `data` instead of `any`?

Reply:
No, it has to be `any`. The `stdin` requests intentionally do not need to be understood by the service worker or service worker manager, they just pass them through. This is so that clients of this functionality such as the Terminal can pass through whatever they need without JupyterLite understanding it. The alternative is that we define a type for `ITerminalStdinRequest` in either the Terminal or JupyterLite and have to import it into the other project, which is not good.

We could be more explicit here and use something like
```ts
data: TDriveRequest<T> | IStdinRequest
```
and have `IStdinRequest` as `any`.

Reply:
Extra documentation on any added in 3268f48.

Reply:
Thanks!